<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CandidatInterfaceClass</class>
 <widget class="QDialog" name="CandidatInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CandidatInterface</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Parti Politique</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nas">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prenom">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_parti">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>49</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_valideCandidat">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>7931</year>
     <month>12</month>
     <day>20</day>
    </datetime>
   </property>
   <property name="maximumDate">
    <date>
     <year>7931</year>
     <month>12</month>
     <day>20</day>
    </date>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="displayFormat">
    <string>dd-MM-yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_numrue">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>49</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nomrue">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ville">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>250</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_code">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prov">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de rue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de naissance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de rue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Code postal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Province</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>951</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ajout d'un candidat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitterBouttonC">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>500</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>QuitterBouttonC</sender>
   <signal>clicked()</signal>
   <receiver>CandidatInterfaceClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
